### PR TITLE
Techs Box

### PR DESCRIPTION
**Pull request by sfernandez11**
# Summary
## Features
* Added tech card with tech data.
* Added empty box behind the tech card for future animation.
## Screenshots
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,5 +13,16 @@
     <img class="title-image" src="assets/sectionTitleImage.svg" alt="image-title"></img>
     <h1 class="title-text">Front End Tools</h1>
   </div>
+  <br>
+  <div class="main-tech-container">
+    <div class="background-tech-container">
+    </div>
+    <div class="data-tech-container">
+      <img class="tech-image" src="assets/react_icon.svg" alt="tech-image"></img>
```
Comment (frparodi) [inline index.html]: Make this tag self-closing ``

File: index.html [hunk 2]
```diff
@@ -13,5 +13,16 @@
     <img class="title-image" src="assets/sectionTitleImage.svg" alt="image-title"></img>
     <h1 class="title-text">Front End Tools</h1>
   </div>
+  <br>
+  <div class="main-tech-container">
+    <div class="background-tech-container">
```
Comment (frparodi) [inline index.html]: Make this div self-closing: `<div class="background-tech-container"/>`

File: scss/techs.scss [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+.main-tech-container {
+  position: relative;
+}
+
+.background-tech-container {
+    border: 1px solid;
+    border-color: $green;
+    border-radius: 5px;
+    height: 300px;
```
Comment (frparodi) [inline scss/techs.scss]: As both card have the same width and height I would extract both of them into variables, and then reference them inside the class block
```
$card-height: 300px;
$card-width: 200px;

.background-tech-container {
    height: $card-height;
    width: $card-width;
}
```

The same for `.data-tech-container`

File: index.html (hunk 1 above)
Comment (sfernandez11) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (sfernandez11) [reply index.html]: Done

File: scss/techs.scss (hunk 3 above)
Comment (sfernandez11) [reply scss/techs.scss]: Done

**Pull request merged**